README.md

```diff
@@ -63,32 +67,34 @@ The following steps are only relevant if you intend to contribute to the reposit
 
 - Install development dependencies (optionally skipping Lockfile creation):
 
-	  pipenv install --dev --skip-lock
+	    pipenv install --dev --skip-lock
```

Comment:
not sure the space is needed.

- Before:

 	  pipenv install --dev --skip-lock

- After:

	    pipenv install --dev --skip-lock

Reply:
It shows as correct in Sublime text ...

Reply:
Yes it happens, the same for the PyCharm plugin

Reply:
see below for the discussion